packages/discovery-provider/plugins/pedalboard/apps/solana-relay/src/middleware/assertRateLimiter.test.ts

```diff
@@ -0,0 +1,70 @@
+import { afterAll, beforeAll, beforeEach, describe, expect, it } from "vitest";
+import { RateLimiter } from "./rateLimiter";
+import { config } from "../config";
+import { getRedisConnection } from "../redis";
+
+beforeEach(async () => {
+    // run with npm run test with audius-compose up
+    config.redisUrl = "redis://0.0.0.0:6379/00"
+    const redis = await getRedisConnection()
+    let cursor = '0';
+    do {
+        for await (const key of redis.scanIterator()) {
+            if (key.startsWith('listens-rate-limit')) {
+                await redis.del(key)
```

Comment:
I'm not at all familiar with how redis works. Is there not a way to bulk delete all the keys in one call instead of iterating on a cursor?

Reply:
not really since all the keys are different

Reply:
at least i don't think so, but this is just a test